src/proxy-api/proxy-api.service.ts

```diff
@@ -57,8 +58,10 @@ export class ProxyApiService {
     const parseResults: KonviwContent[] = data.results.map(
       (doc: ResultsContent) => {
         this.context.Init(spaceKey, doc.content.id);
+        this.context.setHtmlBody(doc.content.body.view.value);
         const atlassianIadcRegEx = new RegExp(`${baseURL}/wiki/`);
-        parseHeaderBlog(doc.content.body.view.value)(this.context);
+        parseHeaderBlog()(this.context);
+        // getFirstExcerpt()(this.context);
```

Comment:
I can't see anywhere in the code that `getFirstExcerpt()` is called - is this commented out line intended?

Reply:
Yes, I did not activate this yet, because I wanted to discuss in a future sprint with you and the team, regarding the full search strategy and how much content we collect to make the API useful. While with more content slower.
So far the partseHeaderBlog is retrieving the excerpt and blog image which was the original intend. My idea was to make it more generic, but to come in a future sprint.